src/calledit.c

```diff
@@ -161,16 +161,13 @@ void calledit(void) {
 		if (strlen(hiscall) + 1 == 12)
 		    break;	// leave insert mode
 
-		if (((i >= 65) && (i <= 90)) || ((i >= 47) && (i <= 57))) {
```

Comment:
No. Please have a look at line 157. That test is already done there.

Reply:
OK, I see. (line 152)
BTW we are just lucky that `/` and `0-9` can be covered by a single ASCII range... Took me some time to figure it out.
If it were
```
 if ((i >= 'A' && i <= 'Z') || (i >= '0' && i <= '9') || i == '/') {
```
it would have been clearer to me.
(comment has a typo: _0_-9)

Reply:
Really old code here. If we touch the code to drop the extra check we should also clean up that test

Reply:
I stared at that for a good while and didn't catch the 1-9 in the comment and I have a 0 in my call!

While this is an example of a simple test using ASCII, I do wonder if we should be using standard library functions or glib functions like `g_ascii_isalnum ()` or `g_ascii_toupper ()`?  In many cases it seems like overkill when a simple `i = i - 32;` does the same thing.  Of course, should the project ever adopt using Unicode beyond the ASCII character set then all of this would need to be revisited.  In new code the use of library functions should be encouraged, IMO.